### DISCUSSION TITLE
How to limit Jupyter Notebook cell height for all outputs in VS Code?

### DISCUSSION THREAD

**Discussion by pehkawn** [microsoft/vscode-jupyter#14113] [Questions and Answers]
Sometimes the output from a code cell will be very large, which makes it impractical to display the entire output. According to [this answer](https://stackoverflow.com/a/69516283/9434803), I tried changing the setting "Notebook>Output: Text Line Limit" (`"notebook.output.textLineLimit": 5`). However, this seems to only limit the cell size of a pure text based output, and not outputs such as matrices, dataframes or other spreadsheet-like outputs (e.g. from the `View() function in R.) 

(NOTE: The cell code is written in R, although I wouldn't presume this to be relevant for this particular issue.)

###### GUI
[![enter image description here][1]][1]

###### (Alternatively) `settings.json`:

```json
"notebook.output.scrolling": true
"notebook.output.textLineLimit": 5
```

###### Text output
Here the cell is limited to 5 lines, with a vertical scroll bar, as defined in the settings:
[![enter image description here][2]][2]

..but not for matrices, dataframes and spreadsheet-like objects.
###### Dataframe
[![enter image description here][3]][3]
###### Output from `View()`
[![enter image description here][4]][4]

#### My question therefore is:

Is it possible to limit cell height for other outputs than text?

  [1]: https://i.stack.imgur.com/rerWY.png
  [2]: https://i.stack.imgur.com/isbX3.png
  [3]: https://i.stack.imgur.com/W1Dry.png
  [4]: https://i.stack.imgur.com/dDGc1.png

**Comment by DonJayamanne** [comment]
@amunger /cc

**Comment by amunger** [comment]
You are correct in that those settings are only supported for text stream outputs, not plaintext or HTML. I've created https://github.com/microsoft/vscode/issues/190426 to track adding that feature.

**Reply by pehkawn** [reply]
Glad to hear it. =)
